siliconcompiler/server.py

```diff
@@ -407,6 +319,36 @@ def auth_password(self, username, password):
             return False
         return (password == self.user_keys[username]['password'])
 
+    def __handle_request(self, request):
```

Comment:
It looks like this function is for common input validation and error-checking, but "handle_request" sounds like a general-purpose method. Maybe rename it to something like `__check_request` or `__verify_request`?